src/plugins/chart_expressions/expression_metric/public/components/metric_vis.tsx

```diff
@@ -336,9 +336,11 @@ export const MetricVis = ({
   const scrollContainerRef = useRef<HTMLDivElement>(null);
   const scrollDimensions = useResizeObserver(scrollContainerRef.current);
 
+  const baseTheme = getThemeService().useChartsBaseTheme();
+
   const {
     metric: { minHeight },
-  } = getThemeService().useChartsBaseTheme();
+  } = baseTheme;
```

Comment:
One note with this is that if eui happens to override this property in `getThemeService().useChartsTheme()` (i.e. [here](https://github.com/elastic/eui/blob/a4477600884b2be7d75a11b87e783fbe08691557/src/themes/charts/themes.ts#L27)) at some point in the future, this logic would break. So something like below would fix this.

```ts
const minHeight = chartTheme?.metric?.minHeight ?? baseTheme.metric.minHeight;
```

> _P.S. Sorry again for the complexity 😔 , also I couldn't write this as a GH code suggestion._

Reply:
Thanks, @nickofthyme ! Will update